internal/kafka/command_link_describe.go

```diff
@@ -97,38 +96,39 @@ func (c *linkCommand) describe(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return err
 		}
-		err = table.Print()
-		if err != nil {
+		if err := table.Print(); err != nil {
 			return err
 		}
-		taskOuts := describeOut.Tasks
-		if len(taskOuts) > 1 {
-			list := output.NewList(cmd)
-			for i := range taskOuts {
-				t := taskOuts[i]
-				var errsStr bytes.Buffer
-				for i := range t.Errors {
-					eo := t.Errors[i]
-					errsStr.WriteString("Error code: ")
-					errsStr.WriteString(eo.ErrorCode)
-					errsStr.WriteString(" ")
-					errsStr.WriteString("Error message: ")
-					errsStr.WriteString(eo.ErrorMessage)
-					if i < len(t.Errors)-1 {
-						errsStr.WriteString(",")
-					}
-				}
-				list.Add(&humanTaskOut{
-					TaskName: t.TaskName,
-					State:    t.State,
-					Errors:   errsStr.String(),
-				})
+		return printHumanTaskOuts(cmd, describeOut.Tasks)
+	}
+}
+
+func printHumanTaskOuts(cmd *cobra.Command, taskOuts []serializedTaskOut) error {
+	if len(taskOuts) == 0 {
+		return nil
+	}
+	list := output.NewList(cmd)
+	for i := range taskOuts {
+		t := taskOuts[i]
+		var errsStr bytes.Buffer
+		for i := range t.Errors {
+			eo := t.Errors[i]
+			errsStr.WriteString("Error Code: ")
+			errsStr.WriteString(eo.ErrorCode)
+			errsStr.WriteString(" ")
+			errsStr.WriteString("Error Message: ")
+			errsStr.WriteString(eo.ErrorMessage)
+			if i < len(t.Errors)-1 {
+				errsStr.WriteString(",")
```

Comment:
How about we use `strings.Join()` instead?

Reply:
Done.